### PR TITLE
mirrors: add tor instructions

### PR DESCRIPTION
**Pull request by andkem**
Adds instructions on using XBPS with Tor.

### CODE REVIEW

File: src/maintenance/repositories/mirrors/tor.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+or as a system service:
+
+```
+# ln -s /etc/sv/tor /var/service
+```
+
+By default, Tor will act as a client and open a SOCKS5 proxy on TCP port 9050 on
+localhost.
+
+### Making XBPS connect via the SCOKS proxy
```
Comment (Vaelatern) [inline src/maintenance/repositories/mirrors/tor.md]: SOCKS

Comment (andkem) [reply src/maintenance/repositories/mirrors/tor.md]: Fixed

File: src/maintenance/repositories/mirrors/tor.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+### Security consideration
+
+It is advisable to set `SOCKS_PROXY` automatically in your environment if you
+are using an onion. If the setting is missing, a DNS query for the name of the
+hidden service will leak to the configured DNS server.
+
+To automatically set the environment variable, add it to `/etc/profile`:
+
+```
+# echo "export SOCKS_PROXY=\"socks5://127.0.0.1:9050\"" >> /etc/profile
```
Comment (the-maldridge) [inline src/maintenance/repositories/mirrors/tor.md]: I'd prefer if this went into a file in /etc/profile.d/

Comment (andkem) [reply src/maintenance/repositories/mirrors/tor.md]: Is the pushed change acceptable? I don't know how easy the here document syntax is to read.

**Pull request merged**